Amazon.QLDB.Driver.IntegrationTests/Amazon.QLDB.Driver.IntegrationTests.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="NLog" Version="4.7.6" />
     <PackageReference Include="AWSSDK.QLDB" Version="3.5.0.57" />
     <PackageReference Include="AWSSDK.QLDBSession" Version="3.5.2" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0.0" />
```

Comment:
Why do we need this? The package states:

> Provides the IAsyncEnumerable<T> and IAsyncDisposable interfaces and helper types for .NET Standard 2.0. This package is not required starting with .NET Standard 2.1 and .NET Core 3.0.

This integration test project is targeting 

>     <TargetFrameworks>netcoreapp2.1;netcoreapp3.1;net48</TargetFrameworks>

Reply:
The build is broken without it.

Reply:
We need to understand why that is the case.

Reply:
I understand; it also left me scratching my head.  But the compiler insists.  The same is encountered by Justin and Arman, I believe.

Reply:
What is the compiler error when you don't include that?

Reply:
![image](https://user-images.githubusercontent.com/62349012/108745476-5dd6c600-74f8-11eb-98ee-4f36dd8f62fd.png)

Reply:
It says "The type 'IAsyncEnumerable<>' is defined in an assembly that is not referenced.  You must add a reference to assembly 'Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0..."
"Asynchronous foreach statement cannot operate on variables of type 'IAsyncResult' because 'IAsyncResult' does not contain a suitable public instance or extension definition for 'GetAsyncEnumerator' (CS8411)"

Reply:
I think this dependency needs to be in the driver, We are referencing `IAsyncEnumerable` in [IAsyncResult](https://github.com/justing-bq/amazon-qldb-driver-dotnet/blob/async-driver-dev/Amazon.QLDB.Driver/result/IAsyncResult.cs#L19)

Also we should set the minimum version to v1.0.0

Thoughts?

Reply:
It doesn't build with v1.0.0. It requires 1.1.0 to build.

Reply:
Ok, I see the SDK is using IAsyncEnumerable v1.1.0

```
/Users/byronlin/projects/github-customer-fork/justin/amazon-qldb-driver-dotnet/Amazon.QLDB.Driver/Amazon.QLDB.Driver.csproj: Error NU1605: Detected package downgrade: Microsoft.Bcl.AsyncInterfaces from 1.1.0 to 1.0.0. Reference the package directly from the project to select a different version. 
 Amazon.QLDB.Driver -> AWSSDK.QLDBSession 3.5.2 -> AWSSDK.Core 3.5.2.9 -> Microsoft.Bcl.AsyncInterfaces (>= 1.1.0) 
 Amazon.QLDB.Driver -> Microsoft.Bcl.AsyncInterfaces (>= 1.0.0) (NU1605) (Amazon.QLDB.Driver)
```

Reply:
@Dominitio please make the change as described above ☝️

Reply:
Addressed